File: Challenge-Sander-Meijer/Angular-FrontEnd/src/app/dtos/work-result-dto.ts
export class WorkResultDto {

    correct: boolean;
    excerciseId: number;
    domain: string;
    learningObjective: string;

    constructor(data: {
        correct: boolean,
        excerciseId: number,
        domain: string,
        learningObjective: string
    }) {
        this.correct = data.correct;
        this.excerciseId = data.excerciseId;
        this.domain = data.domain;
        this.learningObjective = data.learningObjective;
    }
}

File: Challenge-Sander-Meijer/NetCore-BackEnd/src/NetCore-BackEnd/Models/Dto/WorkResultDto.cs
namespace NetCore.BackEnd.Models.Dto
{
	public class WorkResultDto
	{
		public bool Correct { get; set; }

		public int ExcerciseId { get; set; }

		public string Domain { get; set; }

		public string LearningObjective { get; set; }
	}
}